can/bus.py

```diff
@@ -146,4 +150,22 @@ def shutdown(self):
         """
         self.flush_tx_buffer()
 
+    @property
+    @abc.abstractmethod
```

Comment:
You need to remove the `abc.` here. Shouldn't the setter be marked as abstract as well?